client-spark/spark2/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -199,6 +203,23 @@ public <K, V, C> ShuffleHandle registerShuffle(int shuffleId, int numMaps, Shuff
       LOG.info("Generate application id used in rss: " + appId);
     }
 
+    String storageType = sparkConf.get(RssClientConfig.RSS_STORAGE_TYPE);
```

Comment:
Is there any chance to put these codes into common module to avoid making change in both spark2 and spark3 module.

Reply:
ok